tests/L0/run_instance_norm_nvfuser/test_instance_norm_nvfuser.py

```diff
@@ -0,0 +1,69 @@
+import itertools
+import unittest
+
+import torch
+
+import apex
+from apex.normalization import InstanceNorm3dNVFuser
+
+class TestInstanceNormNVFuser(unittest.TestCase):
+    dtype = torch.float
+    track_running_stats = False
+    channels_last = False
+    affine = False
+    batch_size = 5
+    channel_size = 7
+    spatial_size = 3
+
+    def setUp(self):
```

Comment:
[nit] I think it's better to rename this method given how this is called.
`setUp`/`tearDown` is automatically called before/after each method starting with `test`, see [docs](https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp) for the details.